content/docs/operating/security.md

```diff
@@ -177,6 +186,33 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters.
+
+The Go projects will share the same TLS library, which will be based on the
+Go vanilla TLS library. We keep [Go default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+support only TLSv1.2 and higher.
+
+TLS will be added to Java exporters in the future.
+
+If you have special TLS needs, like a different cipher suite or older TLS
+version, the TLS support on the projects should enable you to build a secure
```

Comment:
This reads oddly. Maybe:

the TLS support enables you to

Reply:
done